packages/permissions/src/utils/has-permissions.ts

```diff
@@ -234,43 +236,52 @@ const getDataFenceByPermissionGroup = (
   return null;
 };
 
-export const hasAppliedDataFence = (options: TOptionsForAppliedDataFence) =>
+export const hasAppliedDataFence = (options: TOptionsForAppliedDataFence) => {
+  // If user has `*Manage` as their `appliedPermission`, we bypass `dataFence`.
+  const hasAppliedManagePermission = options.demandedPermissions.some(
+    demandedPermission =>
+      hasManagePermission(demandedPermission, options.actualPermissions)
```

Comment:
I commited with `--no-verify` because I don't really know how to deal with this.

![image](https://user-images.githubusercontent.com/462507/67014940-fa63a980-f0f5-11e9-9556-b2eb7aea3507.png)

Reply:
The message explains well what is happening.

Basically `hasManagePermission` expects a `actualPermissions: TPermissions` but `options.actualPermissions` is of type `TPermissions | null`.

Therefore, I suppose you would do something like this:

```js
demandedPermission => {
  if (options.actualPermissions) {
    return hasManagePermission(demandedPermission, options.actualPermissions);
  }
  return false;
}
```

Reply:
aaah. thanks, bud!

Reply:
fixed ;)